src/sparseml/base.py

```diff
@@ -255,6 +256,9 @@ def check_version(
     :param alternate_package_names: List of alternate names to look for the package
         under if package_name is not found. Useful for nightly builds.
     :type alternate_package_names: Optional[List[str]]
+    :param extra_error_message: optional string to append to error message if error
+        is raised
+    :type extra_error_message: Optional[str]
```

Comment:
Is this a new standard we're following? We already have the types annotated in signature, is this required?

Reply:
not a standard - just keeping consistent with the existing style in this docstring